src/main/java/org/broadinstitute/consent/http/service/LibraryCardService.java

```diff
@@ -37,18 +37,14 @@ public LibraryCardService(LibraryCardDAO libraryCardDAO, InstitutionDAO institut
   public LibraryCard createLibraryCard(LibraryCard libraryCard, User user) {
     throwIfNull(libraryCard);
     boolean isAdmin = checkIsAdmin(user);
-    //If user is not an admin, use user's institutionId rather than the value provided in the payload
-    if (!isAdmin && !libraryCard.getInstitutionId().equals(user.getInstitutionId())) {
-      throw new BadRequestException("Card payload not valid");
-    }
     checkIfCardExists(libraryCard);
     processUserOnNewLC(libraryCard);
-    checkForValidInstitution(libraryCard.getInstitutionId(), libraryCard.getUserEmail());
+    if (!isAdmin) {
+      checkForValidInstitution(user.getInstitutionId(), libraryCard.getUserEmail());
+    }
```

Comment:
This is the one functional change in this PR. This ensures that SOs have to have the same institution as the user for whom the LC is being created.

Reply:
I don't think this is a functional change. Before the code validated that the SO and LC had the same institution, then validated that the LC and the user had the same institution. So it was also verifying that the SO and user had the same institution.